mlflow/models/evaluation/default_evaluator.py

```diff
@@ -69,6 +70,23 @@
 _LATENCY_METRIC_NAME = "latency"
 
 
+def _handle_plotting_errors(exception_type: Type[Exception] = AttributeError, error_message=""):
```

Comment:
What about making this to context manager like `with suppress_plotting_errors` so we can apply to all other `sk_metrics.xxx` call not only custom function? That way we can avoid defining error messages and catch logic multiple places.